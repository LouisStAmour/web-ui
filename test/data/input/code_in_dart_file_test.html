<!doctype html>
<!--
   Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
   for details. All rights reserved. Use of this source code is governed by a
   BSD-style license that can be found in the LICENSE file.
-->
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <link rel="components" href="code_in_dart_file_component.html">
  <script type="application/javascript" src="testing.js"></script>
</head>
<body>
  <div is="my-component"></div>
  <script type="application/dart">
    import 'dart:html';
    main() {
      window.postMessage('done', '*');
    }
  </script>
</body>
</html>
