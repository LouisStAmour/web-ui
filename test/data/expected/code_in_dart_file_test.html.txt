CONSOLE MESSAGE: hello world!
Content-Type: text/plain
<html lang="en"><head><style>template { display: none; }</style>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <link rel="import" href="../input/code_in_dart_file_component.html">
  <script src="../input/packages/web_ui/testing/testing.js"></script>
</head>
<body>
  <div is="my-component"><shadow-root>    <div>Expect 'hello': hello</div>
  </shadow-root></div>



<script type="text/javascript" src="packages/shadow_dom/shadow_dom.debug.js"></script>
<script type="application/dart" src="code_in_dart_file_test.html_bootstrap.dart"></script></body></html>
